Makefile.core.mk

```diff
@@ -403,6 +403,10 @@ endif
 operator-name:
 	sed -i "s/\(projectName:\).*/\1 ${OPERATOR_NAME}/g" PROJECT
 
+.PHONY: operator-chart
+operator-chart:
```

Comment:
Do we want to also overwrite version in chart/values.yaml? And maybe even version and appVersion in chart/Chart.yaml?

Reply:
also done!